Chinese/Donggewudi/Donggeheiheihei.md

```diff
@@ -0,0 +1 @@
+孙悟空变成了东哥，就此解放了压抑数百年的欲望，看见石头就想嘿嘿嘿
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.